scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -766,16 +766,10 @@ static s32 pick_idle_cpu_from(const struct cpumask *cand_cpumask, s32 prev_cpu,
 	 * partially idle @prev_cpu.
 	 */
 	if (smt_enabled) {
-
-		// if layer is prev_over_idle_core, try prev
-		if (prev_in_cand && layer->prev_over_idle_core &&
```

Comment:
Do you want `prev_over_idle_core` to work when SMT is disabled? It seems to ignore that case, is that intended?

Reply:
yep, prev_over_idle_core prefers previous CPU even if SMT sibling is busy.